app/pages/pastebin-saved-paste-page.js

```diff
@@ -0,0 +1,33 @@
+const Page = require('./page')
+const { By } = require('selenium-webdriver');
+
+class PastebinSavedPastePage extends Page {
+
+  constructor() {
+    super()
+
+    this.actualSyntaxHighlighting = By.xpath('/html/body/div[1]/div[2]/div[1]/div[2]/div[3]/div[1]/div[1]/a')
```

Comment:
Can be replaced by XPath with value `'//div[@class = "left"]/a'` or css with value `'div.left'`.